packages/react-components/src/trace-explorer/trace-explorer-views-widget.tsx

```diff
@@ -86,11 +86,10 @@ export class ReactAvailableViewsWidget extends React.Component<ReactAvailableVie
             outputDescription = output.description;
         }
         let traceContainerClassName = 'outputs-list-container';
-        if (props.index === this._lastSelectedOutputIndex) {
+        if (props.index === this.state.lastSelectedOutputIndex) {
             traceContainerClassName = traceContainerClassName + ' theia-mod-selected';
         }
         return <div className={traceContainerClassName}
-            title={outputName + ': ' + outputDescription}
```

Comment:
If this was added by mistake in previous commit, that commit should be amended to remove it.